owncloudApp/src/main/java/com/owncloud/android/workers/RemoveSourceFileWorker.kt

```diff
@@ -0,0 +1,62 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author Aitor Ballesteros Pavón
+ *
+ * Copyright (C) 2024 ownCloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.owncloud.android.workers
+
+import android.content.Context
+import android.net.Uri
+import androidx.core.net.toUri
+import androidx.documentfile.provider.DocumentFile
+import androidx.work.CoroutineWorker
+import androidx.work.WorkerParameters
+import com.owncloud.android.workers.UploadFileFromContentUriWorker.Companion.KEY_PARAM_CONTENT_URI
+import org.koin.core.component.KoinComponent
+import timber.log.Timber
+
+class RemoveSourceFileWorker(
+    private val appContext: Context,
+    private val workerParameters: WorkerParameters
+) : CoroutineWorker(
+    appContext,
+    workerParameters
+), KoinComponent {
+
+    private lateinit var contentUri: Uri
+
+    override suspend fun doWork(): Result {
+        if (!areParametersValid()) return Result.failure()
```

Comment:
I think is more clear here to put only one return and add an if else. 
Something like this:
`return if(a==b){a}else{b}`

Reply:
Well, in this case it seems clear to me, we usually follow this structure when we pass parameters to a worker. We only want it to fail if the parameters are not valid. If they are valid we want it to continue with the following lines of code.